tensorflow_datasets/core/visualization.py

```diff
@@ -66,17 +70,13 @@ def show_examples(ds_info, ds, rows=3, cols=3, plot_scale=3., image_key=None):
   plt = lazy_imports_lib.lazy_imports.matplotlib.pyplot
 
   if not image_key:
-    # Infer the image and label keys
+    #Infer the image and label keys
     image_keys = [
         k for k, feature in ds_info.features.items()
         if isinstance(feature, features_lib.Image)
-    ]
-
-    if not image_keys:
-      raise ValueError(
-          "Visualisation not supported for dataset `{}`. Was not able to "
-          "auto-infer image.".format(ds_info.name))
-
```

Comment:
Why removing these, I think we should print a message if provided data is video or text. Maybe you can add param like audio key and use `if not image_keys and not audio_keys` then again check for `if not image_keys` so print this message else print shows error like visualization is supports only for audio and mage dataset